source/main/MainThread.cpp

```diff
@@ -94,7 +94,8 @@ MainThread::MainThread():
 	m_exit_loop_requested(false),
 	m_base_resource_loaded(false),
 	m_application_state(Application::STATE_NONE),
-	m_next_application_state(Application::STATE_NONE)
+	m_next_application_state(Application::STATE_NONE),
+    m_is_mumble_created(false)

```

Comment:
I know :blush: It's a temporary state. 

The source is historically mostly-tabs (mixed already), but we agreed to make it spaces-only (https://github.com/RigsOfRods/rigs-of-rods/issues/839).

When all PRs are in and 0.4.7 is out, I'll do a full-scale reformat. Doing it now would break PRs. At the same time, some files are spaces- already and I can't be bothered to keep switching my settings.

Sorry.